metabase/metabase
# Trend charts — Compare to another column

Resolves #35553

Adds "another column" comparison type for trend charts

### How to verify

1. New > Question > Raw Data > Sample Database > Products
2. Summarize Count, Min of Price by Created At
3. Set visualization to "trend"
4. Visualization settings > Data tab > Comparisons > Value from another column > Pick "Min of Price"
5. Try different values for "Label" and "Column" and ensure the chart calculates and displays the change as expected

### Demo

https://github.com/metabase/metabase/assets/17258145/51dc5320-7c66-4d92-8a18-7d3063c57d8d
